plugins/module_utils/vmware.py

```diff
@@ -50,8 +50,63 @@ class TaskError(Exception):
     def __init__(self, *args, **kwargs):
         super(TaskError, self).__init__(*args, **kwargs)
 
+def check_answer_question_status(vm):
+    """Check whether locked a virtual machine.
 
-def wait_for_task(task, max_backoff=64, timeout=3600):
+    Args:
+        vm: Virtual machine management object
+
+    Returns: bool
+    """
+    if hasattr(vm, "runtime") and vm.runtime.question:
+        return True
+
+    return False
+
+def make_answer_response(vm, answers):
```

Comment:
```suggestion

def make_answer_response(vm, answers):
```

linters: E302 expected 2 blank lines, found 1